get_subscription.py

```diff
@@ -1,3 +1,22 @@
+# -*- Mode:python; coding: utf-8; c-file-style:"gnu"; indent-tabs-mode:nil -*- */
+#
+# Copyright (C) 2016 Endless Computers INC.
```

Comment:
Nitpick: ©

I think our legal entity is ‘Endless Mobile, Inc.’, so you should probably use that. At least, that’s what’s on the website and in the copyrights for things like `eos-desktop`.